stable/mission-control/UPGRADE_NOTES.md

```diff
@@ -0,0 +1,44 @@
+# JFrog Mission-Control Chart Upgrade Notes
+This file describes special upgrade notes needed at specific versions
+
+## Upgrade from 1.x,2x to 3.X (Chart Versions)
```

Comment:
@rahulsadanandan we may need to add `4.x charts` after https://github.com/jfrog/charts/pull/976 merge

Reply:
Its done.